### DISCUSSION TITLE
Add the ability to make normal save files (and use them when loading the game)

### DISCUSSION THREAD

**Discussion by oswald641** [arianrhodsandlot/retroassembly#15] [Ideas]
I found this project today and I think it's awesome, the only thing that I don't like so far is the fact that I can't find a way to import/export save files from other emulators, only save states.
This also means whenever I boot up a game I have to load a save state because there isn't any save loaded yet, though it's not a big problem, keep up the good work

**Comment by arianrhodsandlot** [comment]
Thanks for your trying! These seem to be two separate problems.

For the first question, do you mean importing a in-game save by uploading a srm file？ Implementing that seems need a lot of hard work. For now, I suggest you to convert your srm files into retroarch save states by your self and then copy them into the automatically generated "retro-assembly" directory inside your ROM directory.

For the second question, that's exactly what I'm also suffering. I'm planning to add a "load the latest state automatically on started" option in the future...when I have enough spare time.
